Source/WebCore/Modules/mediastream/UserMediaController.cpp

```diff
@@ -80,6 +80,7 @@ void UserMediaController::logEnumerateDevicesDenial(Document& document)
         window->printErrorMessage("Not allowed to call enumerateDevices."_s);
 }
 
+#if ENABLE(MEDIA_SESSION)
```

Comment:
Instead of ifdefing entire methods i suspect it might be simpler to ifdef only their body. Usually the former is done if the return type and/or arguments are feature-specific.